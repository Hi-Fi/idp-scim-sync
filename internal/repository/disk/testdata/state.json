{
  "schemaVersion": "",
  "codeVersion": "",
  "lastSync": "2021-09-25T20:49:46+02:00",
  "hashCode": "hashCode",
  "resources": {
    "groups": {
      "items": 1,
      "hashCode": "123456789",
      "resources": [
        {
          "id": "1",
          "name": "group 1",
          "email": "group.1@mail.com",
          "hashCode": "123456789"
        }
      ]
    },
    "users": {
      "items": 1,
      "hashCode": "hashCode",
      "resources": [
        {
          "id": "1",
          "name": {
            "familyName": "1",
            "givenName": "user"
          },
          "displayName": "user 1",
          "active": false,
          "email": "user.1@mail.com",
          "hashCode": "123456789"
        }
      ]
    },
    "groupsUsers": {
      "items": 1,
      "hashCode": "123456789",
      "resources": [
        {
          "items": 1,
          "hashCode": "123456789",
          "group": {
            "id": "1",
            "name": "group 1",
            "email": "group.1@mail.com",
            "hashCode": "123456789"
          },
          "resources": [
            {
              "id": "1",
              "name": {
                "familyName": "1",
                "givenName": "user"
              },
              "displayName": "user 1",
              "active": false,
              "email": "user.1@mail.com",
              "hashCode": "123456789"
            }
          ]
        }
      ]
    }
  }
}