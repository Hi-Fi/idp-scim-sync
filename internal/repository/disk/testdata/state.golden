{"lastSync":"2021-09-25T20:49:46+02:00","hashCode":"123456789","resources":{"groups":{},"users":{},"groupsUsers":{}}}